easybuild/easyconfigs/g/gompi/gompi-2021b.eb

```diff
@@ -0,0 +1,20 @@
+easyblock = 'Toolchain'
+
+name = 'gompi'
+version = '2021b'
```

Comment:
Should we name this `2021.07` until we're confident with the version choices?

Reply:
Yes. Done in d6f90fc and 3485534